runtime/common/src/lib.rs

```diff
@@ -131,6 +131,10 @@ pub mod types {
 	/// Rate with 27 precision fixed point decimal
 	pub type Rate = crate::fixed_point::Rate;
 
+	pub type MaxTokenNameLength = u32;
```

Comment:
Could you document these type aliases? Out of context it's not trivial to know what they refer to.